features/git-town/configuration/configuration_automatic.feature

```diff
@@ -27,13 +27,31 @@ Feature: Automatically running the configuration wizard if Git Town is unconfigu
     | git sync-fork      |
 
 
+  Scenario Outline: Proceeding to configuration upon initial config prompt (using prompt's default)
+    When I run `<COMMAND>` and enter "" and "^C"
+    Then I see "Git Town hasn't been configured for this repository."
+    And I see "Please specify the main dev branch"
```

Comment:
What exactly the question is is irrelevant here. What we really care about is that you got past the question whether you want to run the wizard, and see the first question of the wizard now. 

```cucumber
And I see the first question of the setup wizard
```